pkg/virt-controller/services/rendercontainer.go

```diff
@@ -254,6 +254,12 @@ func wrapExecProbeWithVirtProbe(vmi *v1.VirtualMachineInstance, probe *k8sv1.Pro
 		"--command", originalCommand[0],
 		"--",
 	}
+
+	// skip wrapping if command is already wrapped with "virt-probe" cli tool
+	if originalCommand[0] == wrappedCommand[0] {
+		return
+	}
```

Comment:
I would have run this check even earlier (line 246):
```golang
	if len(originalCommand) < 1 || originalCommand[0] == "virt-probe" {
		return
	}
```

Reply:
fixed as requested